src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -267,6 +273,45 @@ class ExposureSubmissionCoordinator: NSObject, ExposureSubmissionCoordinating, R
 		)
 	}
 
+	private func createWarnOthersViewController() -> ExposureSubmissionWarnOthersViewController {
```

Comment:
I would prefer to just use the current implementation from showWarnOthersScreen() for such a pressing high priority bug and just extract it to a create func.

Reply:
How? `showWarnOthersScreen` does not return anything and the existing flow in the coordinator expects a view controller to be returned. See `getInitialViewController`

Reply:
Well, as I said, by extracting it, just as we did for the other two existing cases.

Reply:
I wanted to refactor the `showWarnOthersScreen` to use THIS controller but did not dare to not break this controller-hell.